lib/oli_web/pow/user_context.ex

```diff
@@ -7,8 +7,10 @@ defmodule OliWeb.Pow.UserContext do
     repo: Oli.Repo,
     user: Oli.Accounts.User
 
-  alias Oli.Repo
   alias Oli.Accounts.User
+  alias Oli.Delivery.Sections
+  alias Oli.Delivery.Sections.Section
+  alias Oli.Repo
```

Comment:
nitpick: any reason to have repo at bottom vs top?

Reply:
I guess it's just to follow an alphabetic order. I generally try to do the same just to choose a criteria but there might be others :)

Reply:
Cool works for me - for some reason I aways put it at top #notsurewhy